store/storetest/channel_store.go

```diff
@@ -6617,13 +6618,287 @@ func testSidebarChannelsMigration(t *testing.T, ss store.Store) {
 	})
 }
 
+func testGetSidebarCategory(t *testing.T, ss store.Store, s SqlSupplier) {
+	t.Run("should return a custom category with its Channels field set", func(t *testing.T) {
+		user := &model.User{Id: model.NewId()}
+		teamId := model.NewId()
+
+		channelId1 := model.NewId()
+		channelId2 := model.NewId()
+		channelId3 := model.NewId()
+
+		appErr := ss.Channel().CreateInitialSidebarCategories(user, teamId)
```

Comment:
The convention that we are following now (with the appError migration) is that AppErrors are declared as err, and errors are declared as nErr. 0/5 if you want to do it here or not, just wanted to let you know.

Reply:
Yeah, I can change that. Most of the old tests use err/appErr as names, but I'm good using the new way of doing it.